doc/nrf/ug_nrf5340.rst

```diff
@@ -174,9 +189,12 @@ In a multi-image build, the image for the application core is the parent image,
 For this to work, the network core image must be explicitly added as a child image to one of the application core images.
 See :ref:`ug_multi_image_defining` for details.
 
-The network sample :ref:`zephyr:bluetooth-hci-rpmsg-sample` is automatically added to all Bluetooth Low Energy samples in the |NCS|.
+The network sample :ref:`zephyr:bluetooth-hci-rpmsg-sample` is automatically added to all Bluetooth Low Energy samples in the |NCS|, and the network sample :ref:`nrf-ieee802154-rpmsg-sample` is automatically added to all Thread and Zigbee samples.
```

Comment:
```suggestion
The network sample :ref:`zephyr:bluetooth-hci-rpmsg-sample` is automatically added to all Bluetooth Low Energy samples in the |NCS|.
```
I would remove 802.15.4, Thead, and Zigbee part from this paragraph. It makes line 193 harder to understand. 802.15.4, Thread, and Zigbee and correctly described in the next paragraph.

Reply:
My bad, will fix asap.